dev-docs/development-process.md

```diff
@@ -106,6 +114,8 @@ To run the Taskcluster UI:
     You will only need to do this when the dependencies change.
   * Run `yarn start` to start the development server.
     It will output a localhost URL on which you can see the site.
+    
+    *Note* : Make sure you have set all the above mentioned environment variables in the shell session.
```

Comment:
This note should be in the previous section. The commands run in `services/web-server` need the environment variables, while the ones run in `ui` do not.

There's already a note in that block, you may want to format both as sentences, like 

> "If you get an error like `readDbUrl is required`, ensure you've set all the environment variables in the shell session."

or both notes like this new style:

> *Note 1*: The pulse note
> *Note 2*: The environment note

Reply:
Sure got it!